lib/src/solver/report.dart

```diff
@@ -375,32 +412,57 @@ $contentHashesDocumentationUrl
         maxAge: Duration(days: 3),
       );
 
+      var messagePrefix = '(';
+      var hasAdvisory = false;
+      if (status.advisoriesUpdated != null) {
```

Comment:
I don't think you need this check. Just trust getAdvisoriesForPackageVersion to return an empty list if there are no advisories.

Reply:
done